libs/extract/src/lib/dropdown/dropdown.ts

```diff
@@ -197,24 +200,38 @@ const update = async (
 const pop = (
   handler: DropdownHandler,
   listener: DropdownListener,
-  isMobile: boolean
+  isMobile: boolean,
+  fixedPlacement?: DropdownPlacements
 ) => {
   if (isMobile && handler.popper) {
     handler.popper.destroy()
     handler.popper = undefined
 
     update(handler, listener, popper(handler.dropdown))
   } else if (!isMobile && !handler.popper) {
-    handler.popper = createPopper(handler.toggler, handler.listbox, {
-      placement: 'bottom-start',
-      modifiers: [
-        {
-          name: 'offset',
-          options: {
-            offset: [0, 4],
-          },
+    const _popperModifiers: Array<{ name: string; options: any }> = [
+      {
+        name: 'offset',
+        options: {
+          offset: [0, 4],
+        },
+      },
+    ]
+
+    if (fixedPlacement) {
+      _popperModifiers.push({
+        name: 'flip',
+        options: {
+          allowedAutoPlacements: [],
         },
-      ],
+      })
+    }
+
+    console.log(_popperModifiers)
```

Comment:
this PR is merging into main, is the console logging here intended?

Reply:
oops, removed 😅